tooling/cli/templates/plugin/build.rs

```diff
@@ -1,4 +1,4 @@
-const COMMANDS: &[&str] = &["ping", "execute"];
```

Comment:
Any specific reason to remove the execute command in this PR?
This breaks the example app UI which invokes the execute command.

Reply:
I meant to remove it and replace it with `ping`, seems like I missed that part, I will update `execute` usages to be `ping` instead

Reply:
should be fixed now